sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -880,34 +881,35 @@ private void setAppPackageInfo(final @NotNull App app, final @NotNull PackageInf
   }
 
   private @Nullable String[] getProGuardUuids() {
-    AssetManager assets = context.getAssets();
-    try {
-      String[] files = assets.list("");
-
-      List<String> listFiles = Arrays.asList(files != null ? files : new String[0]);
-      if (listFiles.contains("sentry-debug-meta.properties")) {
-        try (InputStream is = assets.open("sentry-debug-meta.properties")) {
-          Properties properties = new Properties();
-          properties.load(is);
-
-          String uuid = properties.getProperty("io.sentry.ProguardUuids");
-          if (uuid != null && !uuid.isEmpty()) {
-            return uuid.split("\\|", -1);
-          }
-          options
-              .getLogger()
-              .log(SentryLevel.INFO, "io.sentry.ProguardUuids property was not found.");
-        } catch (IOException e) {
-          options.getLogger().log(SentryLevel.ERROR, "Error getting Proguard UUIDs.", e);
+    final AssetManager assets = context.getAssets();
+
+    try (final InputStream is =
+        new BufferedInputStream(assets.open("sentry-debug-meta.properties"))) {
```

Comment:
`h`: Shouldn't we also call `close` on the assets or put `final AssetManager assets = context.getAssets();` in the try-with-resources block here?

Reply:
I've needed to revert this, `AssetManager` is a single instance per app. and when it gets closed, it's for good.
basically all the context.getResources(...) depends on `AssetManager` and if it's once closed, it's not possible to reuse anymore, this caused crashes when trying to use `Resources` :( TIL.